src/module.c

```diff
@@ -9993,6 +9993,26 @@ int VM_ACLCheckCommandPermissions(ValkeyModuleUser *user, ValkeyModuleString **a
     return VALKEYMODULE_OK;
 }
 
+/* Checks if the command can be executed by the module context user, according to the ACLs
+ * associated with it.
+ *
+ * On success a VALKEYMODULE_OK is returned, otherwise
+ * VALKEYMODULE_ERR is returned and errno is set to the following values:
+ *
+ * * ENOENT: Specified command does not exist.
+ * * EACCES: Command cannot be executed, according to ACL rules; or the user is not authenticated.
+ */
+int VM_ACLCheckCommandPermissionsForContextUser(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int argc) {
```

Comment:
How about renaming "context" -> "current", to match other functions accessing the client in the context: `VM_GetCurrentUserName`, `VM_GetCurrentCommandName`.

```suggestion
int VM_ACLCheckCommandPermissionsForCurrentUser(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int argc) {
```

Reply:
Sure, that makes more sense